src/main/index.ts

```diff
@@ -100,6 +101,36 @@ electron.protocol.registerSchemesAsPrivileged([
   },
 ]);
 
+electron.app.once("ready", (_) => {
+  let sentryBlocked = false;
+
+  ipcMain.on(RepluggedIpcChannels.REGISTER_RELOAD, () => {
+    console.log("sentryblock reset");
+    sentryBlocked = false;
+  });
+
+  session.defaultSession.webRequest.onBeforeRequest(async (details, callback) => {
+    if (!sentryBlocked) {
```

Comment:
This seems like an unnecessarily janky way to do this. For reference, here's how Powercord did it: https://github.com/replugged-org/replugged/blob/legacy/src/Powercord/coremods/no-track/index.js

Reply:
This method also adds delay and overhead.